### DISCUSSION TITLE
Proble with Safe Exam  for iOS 3.0.1

### DISCUSSION THREAD

**Discussion by ari1101** [SafeExamBrowser/seb-mac#204] [General]
good evening
 
In the case of safe test for iOS we have been presented with the problem of the image, we have attached and installed the latest version of safe exam bowser  for iOS 3.0.1, but, the problem persist.

stay tuned, thank
![MicrosoftTeams-image (3)](https://user-images.githubusercontent.com/91102971/162358218-5db9f8cb-1803-4f4a-9307-99175344442b.png)
s

**Comment by danschlet** [comment]
This iOS issue usually happens when SEB crashes (you can see the home screen for a moment and then SEB is restarted automatically). Sometimes selecting Cancel and restarting SEB can help, otherwise you need to restart your device as mentioned in the error message.
